src/domains/chart/components/snapshot-loader.tsx

```diff
@@ -35,6 +35,7 @@ const SnapshotLoader = ({
     dispatch(fetchDataForSnapshotAction.request({
       // properties to be passed to API
       host,
+      context: attributes.id, 
```

Comment:
please note that snapshots are now only available for standalone dashboard (so `context` is not needed now)